projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss

```diff
@@ -115,7 +115,7 @@ select {
 
 #ClearSearchFilter {
     position: absolute;
-    right: 2px;
+    right: -45px;
```

Comment:
Negative orientation for `left/right` styling is also an anti-pattern. Usually indicates a larger issue with the layout of the elements container.

Reply:
will address in my next PR